tests/sentry/integrations/slack/test_message_builder.py

```diff
@@ -1631,18 +1632,44 @@ def setUp(self):
             type=FeedbackGroup.type_id, substatus=GroupSubStatus.NEW
         )
 
+    @freeze_time("2024-02-23")
     @with_feature("organizations:slack-block-kit-improvements")
     def test_get_context(self):
-        # endpoint regression should use Regressed Date
+        # endpoint regression should use Approx Start Time
         context = get_context(self.endpoint_regression_issue)
-        assert "Regressed Date: *1\xa0hour ago*" in context
+        assert f"Approx. Start Time: *{self.one_hour_ago.strftime('%Y-%m-%d %H:%M:%S')}*" in context
 
         # crons don't have context
         assert get_context(self.cron_issue) == ""
 
         # feedback doesn't have context
         assert get_context(self.feedback_issue) == ""
 
+    @with_feature("organizations:slack-block-kit-improvements")
+    def test_get_context_error_user_count(self):
+        event = self.store_event(
+            data={},
+            project_id=self.project.id,
+            assert_no_errors=False,
+        )
+        group = event.group
+        assert group
+
+        context_without_error_user_count = get_context(group)
+        assert (
+            context_without_error_user_count
+            == f"State: *Ongoing*   First Seen: *{time_since(group.first_seen)}*"
+        )
+
+        group.times_seen = 3
+        group.save()
+
+        context_with_error_user_count = get_context(group)
+        assert (
+            context_with_error_user_count
+            == f"Events: *3*   State: *Ongoing*   First Seen: *{time_since(group.first_seen)}*"
```

Comment:
we don't show `None` or `0` values inside the context. since `Users Affected` is `0` here we don't show it.

Reply:
not sure if the logic should be changed for this to only check for `None`